data/forms/child-referral.jsx

```diff
@@ -0,0 +1,565 @@
+export default {
+  title: 'FAST referral form',
+  path: '/form/child-referral/',
+  steps: [
+    {
+      id: 'client-details',
+      title: 'Client Details',
+      components: [
+        {
+          component: 'TextInput',
+          name: 'worker_email',
+          width: '30',
+          label: 'Email address',
+          hint: 'Valid email address',
+          rules: { required: true },
+        },
+        {
+          component: 'Radios',
+          name: 'businessSupport',
+          label:
+            'Are you a member of FAST Business Support at Hackney Council?',
+          options: ['Yes', 'No'],
+          rules: { required: true },
+        },
+      ],
+    },
+    {
+      id: 'aboutFASTBuisnessSupport',
+      title: 'About FAST Business Support team member',
+      components: [
+        {
+          component: 'TextInput',
+          name: 'business_support_name',
+          width: '30',
+          label:
+            'As a member of FAST Business Support, please provide YOUR full name (not that of the referrer).',
+        },
+      ],
+    },
+    {
+      id: 'aboutYourself',
+      title: 'About yourself',
+      components: [
+        {
+          component: 'TextInput',
+          name: 'full_name',
+          width: '30',
+          label: 'Your full name',
+          rules: { required: true },
+        },
+        {
+          component: 'TextInput',
+          name: 'referrer_relationship',
+          width: '30',
+          label: 'Role / relationship to the child or family: ',
+          rules: { required: true },
+        },
+        {
+          component: 'TextInput',
+          name: 'position_agency',
+          width: '30',
+          label: 'Position / agency (if relevant): ',
+        },
+        {
+          component: 'TextInput',
+          name: 'referrer_telephone',
+          width: '30',
+          label: 'Telephone Number',
+          rules: { required: true },
```

Comment:
We should validate that is a phone number, or at least that is a number.
Potentially we could also define the `type` as `tel` (you can find more info here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)

Reply:
Added generic number component, test & storyboard. Used number type instead as tel type allows letters unless you use a regular expression and considering the different types of phone numbers it would be quite a large regex